code/game/objects/effects/mapping_helpers.dm

```diff
@@ -0,0 +1,72 @@
+/obj/effect/baseturf_helper //Set the baseturfs of every turf in the /area/ it is placed.
+	name = "baseturf editor"
+	icon = 'icons/effects/mapping_helpers.dmi'
+	icon_state = ""
+
+	var/baseturf
+
+	layer = POINT_LAYER
+
+/obj/effect/baseturf_helper/Initialize(mapload)
+	. = ..()
+	var/area/thearea = get_area(src)
+	for(var/turf/T in get_area_turfs(thearea, z))
+		replace_baseturf(T)
+	return INITIALIZE_HINT_QDEL
```

Comment:
they use 

```
/obj/effect/baseturf_helper/Initialize(mapload)
	. = ..()
	return INITIALIZE_HINT_LATELOAD

/obj/effect/baseturf_helper/LateInitialize()
	if(!baseturf_to_replace)
		baseturf_to_replace = typecacheof(/turf/open/space)
	else if(!length(baseturf_to_replace))
		baseturf_to_replace = list(baseturf_to_replace = TRUE)
	else if(baseturf_to_replace[baseturf_to_replace[1]] != TRUE) // It's not associative
		var/list/formatted = list()
		for(var/i in baseturf_to_replace)
			formatted[i] = TRUE
		baseturf_to_replace = formatted

	var/area/our_area = get_area(src)
	for(var/i in get_area_turfs(our_area, z))
		replace_baseturf(i)

	qdel(src)
```

We do have the `LateInitialize` system in place.

Reply:
/tg/ somewhat recently updated their baseturf system to support multiple baseturfs. We still only support one baseturf, so this is still using their old system.